app/graphql/generated.gql

```diff
@@ -778,6 +778,44 @@ query ContactsCard {
   }
 }
 
+query ExportCsvSetting($walletIds: [WalletId!]!) {
+  me {
+    id
+    defaultAccount {
+      id
+      csvTransactions(walletIds: $walletIds)
+      __typename
+    }
+    __typename
+  }
+}
+
+query SettingsScreen {
+  me {
+    username
```

Comment:
```suggestion
  me {
    id
    username
```

I think we should always have the id here, otherwise it won't get cache properly

cc @UncleSamtoshi